cmd/kube-apiserver/app/options/options.go

```diff
@@ -150,6 +150,9 @@ func (s *ServerRunOptions) Flags() (fss apiserverflag.NamedFlagSets) {
 	fs.BoolVar(&s.EnableLogsHandler, "enable-logs-handler", s.EnableLogsHandler,
 		"If true, install a /logs handler for the apiserver logs.")
 
+	// Deprecated in release 1.13
+	fs.MarkDeprecated("enable-logs-handler", "This flag will be removed in a future version.")
+
```

Comment:
it seems it is the convention to have the flag declaration and its MarkDeprecated together, without a blank line in between.

Reply:
done